R/pkg/R/DataFrame.R

```diff
@@ -276,6 +276,57 @@ setMethod("names<-",
             }
           })
 
+#' @rdname columns
+#' @name colnames
```

Comment:
In R doc, it will be included under the `columns` page because of `@rdname` notation. https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html